src/Chapter2.hs

```diff
@@ -349,8 +350,11 @@ Implement a function that returns only the first half of a given list.
 "b"
 -}
 -- PUT THE FUNCTION TYPE IN HERE
+firstHalf :: [a] -> [a]
 firstHalf l = error "firstHalf: Not implemented!"
```

Comment:
I see `error` is left in here. But other than that, the implementation is correct :+1: